cmd/bnscli/cmd_sigs.go

```diff
@@ -0,0 +1,114 @@
+package main
+
+import (
+	"encoding/hex"
+	"encoding/json"
+	"errors"
+	"flag"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/iov-one/weave/cmd/bnsd/app"
+	"github.com/iov-one/weave/cmd/bnsd/client"
+	"github.com/iov-one/weave/crypto"
+)
+
+func cmdSignTransaction(
+	input io.Reader,
+	output io.Writer,
+	args []string,
+) error {
+	fl := flag.NewFlagSet("", flag.ExitOnError)
+	fl.Usage = func() {
+		fmt.Fprint(flag.CommandLine.Output(), `
+Sign given transaction. This is decoding a transaction data from standard
+input, adds a signature and writes back to standard output signed transaction
+content.
+
+`)
+		fl.PrintDefaults()
+	}
+	var (
+		tmAddrFl = fl.String("tm", "https://bns.NETWORK.iov.one:443", "Tendermint node address. Use proper NETWORK name.")
+		keyFl    = fl.String("key", "", "Hex encoded, private key that transaction should be signed with.")
```

Comment:
At the least, we should refer to a file here.

https://github.com/iov-one/weave/blob/master/cmd/bnsd/client/keys.go

has some basic functions for working with private keys on disk. Not encrypted or secure yet, but a good place to get this working. We could add an encryption function and password here in the future (I did this for gaia/cosmos-sdk, and worked on the iov-core model, so I can give a number of pointers). Out of scope for this PR, but simply moving to unencrypted file storage (which can be mode 0400 and not visible via `ps`) will be a big security improvement

Reply:
Done.